control_plane/attachment_service/src/persistence.rs

```diff
@@ -75,6 +78,25 @@ pub(crate) enum DatabaseError {
     Logical(String),
 }
 
+#[derive(measured::FixedCardinalityLabel, Clone)]
+pub(crate) enum DatabaseOperation {
+    InsertNode,
```

Comment:
nit: does `measured` automatically snake_case these?  Would prefer to avoid CamelCase in labels